fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -307,6 +321,7 @@ void DLApi::init() {
 	loadClientFunction(&api->transactionReset, lib, fdbCPath, "fdb_transaction_reset");
 	loadClientFunction(&api->transactionCancel, lib, fdbCPath, "fdb_transaction_cancel");
 	loadClientFunction(&api->transactionAddConflictRange, lib, fdbCPath, "fdb_transaction_add_conflict_range");
+	loadClientFunction(&api->transactionGetStorageByteSample, lib, fdbCPath, "fdb_transaction_get_storage_byte_sample", headerVersion >= 700);
```

Comment:
My comment was lost after I applied the suggestion...

So my question was: how and when do I need to increase the API version? @ajbeamon

Reply:
The API version will get increased at some point for every major or minor release. It's sometimes the case that we can change it right before the release, while other times it is necessary in order to properly build a feature that it be increased earlier in the development cycle. Usually, you would do this at a point where you've got to write some code that will only operate at newer API versions and you need to be able to test it.

In this case, none of your code is specifically guarded by the API version, so it will work the same at older versions and it's not necessary to change the API version throughout the codebase yet (though you could). Instead, the reference to API version 700 here acts to filter out any external clients that don't support 700 if the client requests it.